nomad/state/testing.go

```diff
@@ -21,3 +24,70 @@ func TestStateStore(t testing.T) *StateStore {
 	TestInitState(t, state)
 	return state
 }
+
+// CreateTestPlugin is a helper that generates the node + fingerprint results necessary to
+// create a CSIPlugin by directly inserting into the state store. It's exported for use in
+// other test packages
+func CreateTestCSIPlugin(s *StateStore, id string) func() {
```

Comment:
This is just moving from `nomad` to `state` to avoid a dependency cycle using it in `state`

Reply:
:+1: I'm pretty sure I ran into that at one point on another test and punted on fixing it, so thanks for picking this up here.